karma.conf.babel.js

```diff
@@ -0,0 +1,80 @@
+import paths from './paths';
```

Comment:
The current test script just runs karma directly: `"test": "karma start karma.conf.js",`.  The private template repo linked there is not implemented here yet.